voctocore/lib/videomix.py

```diff
@@ -45,9 +45,24 @@ def __init__(self):
 
         self.names = Config.getlist('mix', 'sources')
         self.log.info('Configuring Mixer for %u Sources', len(self.names))
+        
+        self.has_alpha_method = 0
```

Comment:
Shouldn't this be `= False`?
Also on first read I struggled to identify why this information is required this early on, so probably a variable named `has_any_source_alpha_enabled = False` would be better.
*Edit:* I think this logic might not be needed at all, see below